www/templates/html/MediaPlayer.tpl.php

```diff
@@ -6,13 +6,184 @@
 }
 ?>
 
+<?php if($context->media->getTextTracks()): ?>
+<script type="html/template" class="mh_transcript_template">
+
+        <div class="mh-caption-search">
+            <h6 class="wdn-sans-serif wdn-icon-search">
+                Searchable Transcript
+                <div class="wdn-icon-info mh-tooltip hang-right italic" id="privacy-details">
+                    <div>
+                        <ul>
+                            <li>Use the text input to search the transcript. </li>
+                            <li>Click any line to jump to that spot in the video. </li>
+                            <li>Use the icons to the right to toggle between list and paragraph view. </li>
+                        </ul>
+                    </div>
+                </div>
+            </h6>
+            <div class="mh-caption-container">   
+                <label for="mh-parse-caption">Search:</label>
+                <div class="mh-paragraph-icons">
+                    <div class="mh-bullets"></div>
+                    <div class="mh-paragraph"></div>
+                </div>
+                <br>
+                <input id="mh-parse-caption" type="text" class="mh-parse-caption"><div class="mh-caption-close"></div>
+                <ul class="mh-transcript"></ul>
+            </div>
+        </div>
+
+</script>
+<?php endif; ?>
+
 <script type="text/javascript">
     (function () {
         var e = function () {
             <?php if (isset($context->media->id) && $context->media->id) { ?>
             WDN.setPluginParam('mediaelement_wdn', 'options', {
-                success: function (m, v) {
-                    //Playcount
+                success: function (m, v, t) {
+                    var $ = WDN.jQuery;
+                    var $transcript;
+                    var $captionSearch;
+
+                    <?php if($context->media->getTextTracks()): ?>
+
+                        t.container.append(WDN.jQuery(".mh_transcript_template").html());
```

Comment:
The `WDN.jQuery` variable is localized to `$`. You should use that to keep things neat.